server/service.go

```diff
@@ -144,6 +144,16 @@ func (m *BoostService) handleRegisterValidator(w http.ResponseWriter, req *http.
 			http.Error(w, errInvalidSignature.Error(), http.StatusBadRequest)
 			return
 		}
+
+		ok, err := types.VerifySignature(registration.Message, types.DomainBuilder, registration.Message.Pubkey[:], registration.Signature[:])
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusBadRequest)
```

Comment:
we should probably log the error (with `log.Error`), for visibility for the operator